lib/act.js

```diff
@@ -9,8 +9,9 @@ var call;
 var config;
 var widget;
 var cache;
+var debug = require('debug')('fh-mbaas-api:act');
```

Comment:
I'd be in favor of not forking in that case and just documenting how to enable timestamps as @wojta says in the README comment.